tests/request.js

```diff
@@ -0,0 +1,297 @@
+'use strict';
+
+const { PassThrough } = require('stream');
+const sandbox = require('sinon').createSandbox();
```

Comment:
Hace muchas versiones que `sinon` ya es un sandbox y no hay que crear otro:
```suggestion
const sinon = require('sinon');
```